ee/server/service/hosts.go

```diff
@@ -172,6 +172,11 @@ func (svc *Service) UnlockHost(ctx context.Context, hostID uint) (string, error)
 	case lockWipe.IsPendingLock():
 		return "", ctxerr.Wrap(ctx, fleet.NewInvalidArgumentError("host_id", "Host has pending lock request. Host cannot be unlocked until lock is complete."))
 	case lockWipe.IsPendingUnlock():
+		// MacOS machines are unlocked by typing the PIN into the machine. "Unlock" in this case
+		// should just return the PIN as many times as needed.
+		if host.FleetPlatform() == "darwin" {
```

Comment:
To accommodate this change here, I think we need to also add the activity from `enqueueUnlockHostRequest` here as well

Reply:
@gillespi314 good point! decided to just `break` so that we only have to create the activity in one place (the helper func below). I'm refactoring that now so that we create the activity no matter the host OS.